scripts/create-release-branch-workflow.sh

```diff
@@ -0,0 +1,7 @@
+CircleCIToken=$1
+curl -v -u ${CircleCIToken}: -X POST --header "Content-Type: application/json" -d '{
+  "branch": "develop",
+  "parameters": {
+    "create-release": true
```

Comment:
These are the pipeline parameters passed in via the API call to CircleCi. We can easily add a parameter here to change how circleci handles various workflows, jobs, and commands.